uiowa/uiowa
# Academic calendar part deux

Resolves https://github.com/uiowa/uiowa/issues/7854




# How to test

- Sync registrar.uiowa.edu
- Place the Academic Calendar block on https://registrar.uiowa.ddev.site/academic-calendar
- See that the date cards are using the new UIDS component definition/styles.
- See that on initial load there is a spinner that is replaced by results.
- The group by month can be configured to show/hide as well as be selected as default. Show/hide previous works the same even if the group by month is not available.
- All of the filters trigger the necessary fetch and filter.
- If you happen to have VoiceOver on, even though it isn't in-scope, there should be announcements letting the user know that the results are changing along with basic details about what to expect.
- The jquery dependency has been removed, travis is passing.
- The subsessions are showing in the correct order
- The sorting is happening in PHP and is being cached.
- The date is now using a gold border instead of a full gold background.
- Items after the first day in a month do not have a date display circle, grouping them together.
- There is a day of the week display in the subtitle.
- Sessions in the `sessions` dropdown are sorted
- There are no errors when changing the settings of an Academic calendar block after it has already been placed on a page.
